corehq/apps/app_manager/static/app_manager/js/app-notifications.js

```diff
@@ -0,0 +1,14 @@
+/* globals hqDefine moment alert_user */
+hqDefine('app_manager/js/app-notifications.js', function () {
+    function NotifyFunction(userId) {
```

Comment:
Might be a good use of the notification API? https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API/Using_the_Notifications_API